change/@fluentui-contrib-azure-theme-50ec81f2-7b56-4409-ba3e-74f8e5767a9c.json

```diff
@@ -1,6 +1,6 @@
 {
-  "type": "minor",
```

Comment:
note: this would release 1st version as breaking change  -> from `0.0.0` to `0.1.0`

Reply:
Correct, looks good.